logstash.conf.m4

```diff
@@ -29,16 +31,16 @@ output {
 }
 filter {
  ruby {
-   code => "event['timestamp'] /= 1_000_000"
```

Comment:
Only problem I see here is that if we do `event['timestamp'] /= 1_000_000` the timestamp will be in milliseconds, which is really unintuitive. That was the reason I added _ms suffix to the field name. If we want to keep the field name as `timestamp` most logical option here would be to do `event['timestamp'] /= 1_000_000_000` so timestamp would be in seconds. I don't think this is that big issue because everyone will anyway create their own config file.